src/index.js

```diff
@@ -1,14 +1,24 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
+import { ApolloClient, InMemoryCache } from "@apollo/client";
+import { ApolloProvider } from "@apollo/client/react";
 
-ReactDOM.render(
-  <React.StrictMode>
+const API_KEY = '25f50ab3-2426-447f-a4dc-a521a96ec52e'
```

Comment:
In real world it should be set at env level to be able to change it on production environtment but in this scenario this is cool 👍